runtime/common/src/lib.rs

```diff
@@ -124,3 +124,82 @@ parameter_types! {
 /// https://w3f-research.readthedocs.io/en/latest/polkadot/Token%20Economics.html#-2.-slow-adjusting-mechanism
 pub type SlowAdjustingFeeUpdate<R> =
 	TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+pub fn migrate_treasury() {
+	// --- crates.io ---
+	use codec::{Decode, Encode};
+	// --- paritytech ---
+	use frame_support::{migration, StorageHasher, Twox64Concat};
+	// --- darwinia-network ---
+	use darwinia_primitives::*;
+
+	type ProposalIndex = u32;
+
+	const OLD_PREFIX: &[u8] = b"DarwiniaTreasury";
+	const NEW_PREFIX: &[u8] = b"Treasury";
+
+	migration::move_storage_from_pallet(b"ProposalCount", OLD_PREFIX, NEW_PREFIX);
+	log::info!("`ProposalCount` Migrated");
```

Comment:
If there are proposals for `Instance2Treasury` (KTONTreasury), we might lost the count information for `Instance2Treasury`.

`Instance2Treasury` 's ProposalCount looks to zero.